packages/components/grid/src/filter.ts

```diff
@@ -66,10 +68,17 @@ export class GridFilter extends ScopedElementsMixin(LitElement) {
   @property({ attribute: false }) options?: GridFilterOption[];
 
   set value(value: string | string[] | undefined) {
-    this.#value = value;
+    if (this.mode !== 'text') {
+      this.#value = Array.isArray(value) ? value : value?.split(',');
+    } else {
+      this.#value = value;
+    }
+
+    this.active = Array.isArray(this.#value) ? this.#value.length > 0 : !!this.#value;
+    this.requestUpdate('value');
   }
 
-  @property()
+  @property({ type: String })
```

Comment:
I assume the support for filtering on numbers will come later?

Reply:
Good point, but yes :)